docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,186 @@
+You don't need to define certain settings and parameters or create {infer} ingest pipelines.
+
+The recommended way to use semantic search in the {stack} is following the semantic text workflow.
+When you need more control over the settings, you can still use the complete {infer} workflow (refer to  <<semantic-search-inference,this tutorial>> to review the process).
+
+This tutorial uses the `elser` service for demonstration but you can use any service and their supported models offered by the {infer-cap} API.
+
+
+[discrete]
+[[semantic-text-requirements]]
+==== Requirements
+
+To use `semantic_text` field type, you must have an {infer} endpoint deployed in
+your cluster using the <<put-inference-api>>.
+
+
+[discrete]
+[[semantic-text-infer-endpoint]]
+==== Create the {infer} endpoint
+
+Create an inference endpoint by using the <<put-inference-api>>:
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/sparse_embedding/my-elser-model <1>
+{
+  "service": "elser", <2>
+  "service_settings": {
+    "num_allocations": 1,
+    "num_threads": 1
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The task type is `sparse_embedding` in the path as the `elser` service will
+be used and ELSER creates sparse vectors. The `inference_id` is
+`my-elser-model`.
+<2> The `elser` service is used in this example.
+
+
+[discrete]
+[[semantic-text-index-mapping]]
+==== Create the index mapping
+
+The mapping of the destination index - the index that contains the embeddings
+that the model will create based on your input text - must be created. The
+destination index must have a field with the <<semantic-text,`semantic_text`>>
+field type to index the output of the used model.
+
+[source,console]
+------------------------------------------------------------
+PUT semantic-embeddings
+{
+  "mappings": {
+    "properties": {
+      "semantic_text": { <1>
+        "type": "semantic_text", <2>
+        "inference_id": "my-elser-model" <3>
+      },
+      "content": { <4>
+        "type": "text",
+        "copy_to": "semantic_text" <5>
+      }
+    }
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The name of the field to contain the generated tokens.
+<2> The field to contain the tokens is a `semantic_text` field.
+<3> The `inference_id` is the inference endpoint you created in the previous step.
+It will be used to generate the tokens based on the input text.
+Every time you ingest data into the related `semantic_text` field, this endpoint will be used for creating the vector representation of the text.
+<4> The field to store the text reindexed from a source index in the  <<semantic-text-reindex-data,Reindex the data>> step.
+<5> The textual data stored in the `content` field will be copied to `semantic_text` and processed by the {infer} task.
+The `semantic_text` field will store the tokens generated based on the input data.
```

Comment:
This may be a nit, but I thought we wanted to stay away from referring to ELSER output as "tokens". Should we generalize this to "embeddings"?

Reply:
Fair point Mike, ++